projects/abseil.io/package.yml

```diff
@@ -0,0 +1,42 @@
+distributable:
+  url: https://github.com/abseil/abseil-cpp/archive/refs/tags/{{version.raw}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: abseil/abseil-cpp/releases/tags
+
+build:
+  dependencies:
+    cmake.org: ^3
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+
+  script: |
+    cmake -S . -B build \
+      -DCMAKE_CXX_STANDARD=17 \
+      -DBUILD_SHARED_LIBS=ON \
+      -DCMAKE_INSTALL_RPATH={{ prefix }}/bla \
+      -DCMAKE_BINARY_DIR={{ prefix }}/bin \
+      -DABSL_PROPAGATE_CXX_STD=ON \
+      -DCMAKE_INSTALL_PREFIX={{ prefix }} \
+      -DCMAKE_INSTALL_LIBDIR={{ prefix }}/lib \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_FIND_FRAMEWORK=LAST \
```

Comment:
if this is required needs a comment

Reply:
I copied the Homebrew formula. I don't even know what this library does, much less if these flags are required. I might not be the best person to build packages...